docs/userstories/v1/egress-control-in-anp/egress-control-cluster-nodes.md

```diff
@@ -0,0 +1,41 @@
+
+### Summary
+
+1. **As a** cluster administrator **I want** to easily block access from
+cluster workloads to k8s-apiserver.
+2. **As a** cluster administrator **I want** to easily block access from
+cluster workloads to specific ports on cluster nodes.
+
+### Description
+
+Currently the behaviour for policies defined around traffic from cluster
+workloads (non-hostNetworked pods) towards hostNetworked pods or nodes in the
+cluster is undefined. See https://github.com/kubernetes-sigs/network-policy-api/issues/73.
+
+ANP currently supports only east-west traffic and this traffic flow cuts from
+overlay to underlay which makes this part of the egress (south-north) use case.
+
+Let's provide a defined behaviour in ANP to explicitly achieve the use cases.
+
+### Acceptance Criteria
+
+1. If I apply a policy that limits egress access from workloads identified
+by label:A towards k8s-apiserver backends (master nodes at 6443 port), it should
+be able to guarantee that those workloads cannot access the apiserver
+endpoints:targetPort. Even if the master node IPs change, the policy should be
+able to automatically pivot towards the new nodeIPs.
+
+[
+Also, here are a few points that need to be addressed:
+
+1. Since ANP is a new API, define if our current namespace and pod selectors
+consider host-networked pods or not? Or do we want to allow this discretion to the
+CNI plugins?
+2. Can we just add a `nodeSelector` field that can be specially used to select
```

Comment:
might remove this point if we get consensus on the above discussion which makes this point redundant.